MANIFEST.in

```diff
@@ -1,5 +1,6 @@
 # Include docs
 include *.rst LICENSE
+include pyvistaqt/data/pyvista_logo_square.png
```

Comment:
@banesullivan this adds a 147 kB binary to the repo. If you want to pick a better logo and think it can be done quickly, it might be a good idea to do it before this is merged so that the history doesn't have this in it.

Reply:
I won't have time to make a better logo for this... though it is high on my priority list for us to get a new, sleek logo

Reply:
Fair enough. In that case, this PR is good to go from my end